frontend/src/components/auth/Credentials.tsx

```diff
@@ -92,65 +100,120 @@ const Credentials = ({
-            <input
-              type="email"
-              value={email}
-              onChange={(event) => setEmail(event.target.value)}
-              placeholder="username@domain.com"
-            />
-          </div>
-          <div>
-            <input
-              type="password"
-              value={password}
-              onChange={(event) => setPassword(event.target.value)}
-              placeholder="password"
-            />
-          </div>
-          <div>
-            <button
-              className="btn btn-primary"
-              type="button"
-              onClick={onLogInClick}
-            >
-              Log In
-            </button>
-          </div>
-          <GoogleLogin
-            clientId={process.env.REACT_APP_OAUTH_CLIENT_ID || ""}
-            buttonText="Login with Google"
-            onSuccess={(response: GoogleResponse): void => {
-              if ("tokenId" in response) {
-                onGoogleLoginSuccess(response.tokenId);
-              } else {
-                // eslint-disable-next-line no-alert
-                window.alert(response);
-              }
-            }}
-            onFailure={(error: GoogleErrorResponse) =>
-              // eslint-disable-next-line no-alert
-              window.alert(JSON.stringify(error))
-            }
-          />
-        </form>
-        <div>
-          <button
-            className="btn btn-primary"
-            type="button"
-            onClick={onSignUpClick}
+      <Flex h="100vh">
+        <Box w="47%">
+          <Flex
+            position="absolute"
+            top="27%"
+            left="6%"
+            w="36%"
+            h="30%"
+            justifyContent="space-between"
+            direction="column"
           >
-            Sign Up
-          </button>
-        </div>
-      </div>
-    );
+            <Box display="flex"  alignItems="flex-start">
+              <Text 
+                position="absolute" 
+                fontWeight="700" 
+                fontSize="40px" 
+                color="#000000"
+              >
+                Log In
+              </Text>
+            </Box>
+            <Box>
+              <Input 
+                height="7vh" 
+                placeholder="Your email address"
+                fontWeight="400"
+                fontSize="22px"
+                lineHeight="29px"
+                position="absolute"
+                color="#989898"
+                _placeholder={{
+                color: "#989898"
+                }}
+                type="email"
+                value={email}
+                onChange={(event) => setEmail(event.target.value)}
+              />
+            </Box>
+            <Box>
+              <Input 
+                  height="7vh" 
+                  placeholder="Your password"
+                  fontWeight="400"
+                  fontSize="22px"
+                  lineHeight="29px"
+                  position="absolute"
+                  color="#989898"
```

Comment:
The text colour and the placeholder colour should be different. I am pretty sure the text should be "#535353" and the placeholder is "#989898" like you have already.